tests/unit/EnvTest.php

```diff
@@ -471,6 +471,18 @@ public function testGetArray_notFoundWithInvalidDefaultType()
         // Assert
         $this->fail("Should have thrown TypeError on default array.");
     }
+
+    public function testGetArray_notFoundWithNullDefault()
+    {
+        // Arrange
+        $varname = 'TESTGETARRAY_EXISTSWITHINVALIDDEFAULTTYPE';
```

Comment:
is `$varname` supposed to match the function name?

Reply:
Not necessarily, but it would make more sense. Thanks for noticing that.